fluss-server/src/test/java/com/alibaba/fluss/server/metadata/TabletServerMetadataCacheTest.java

```diff
@@ -133,9 +144,164 @@ void testUpdateClusterMetadataRequest() {
+                Arrays.asList(
+                        new BucketMetadata(0, 0, 0, Arrays.asList(0, 1, 2)),
+                        new BucketMetadata(1, NO_LEADER, 0, Arrays.asList(1, 0, 2))));
+
+        // test delete one table.
+        serverMetadataCache.updateClusterMetadata(
+                new ClusterMetadata(
+                        coordinatorServer,
+                        aliveTableServers,
+                        Collections.singletonList(
+                                new TableMetadata(
+                                        TableInfo.of(
+                                                DATA1_TABLE_PATH,
+                                                DELETED_TABLE_ID, // mark this table as
+                                                // deletion.
+                                                1,
+                                                DATA1_TABLE_DESCRIPTOR,
+                                                System.currentTimeMillis(),
+                                                System.currentTimeMillis()),
+                                        Collections.singletonList(
+                                                new BucketMetadata(
+                                                        1, 1, 0, Arrays.asList(1, 0, 2))))),
+                        Collections.emptyList()));
+        assertThat(serverMetadataCache.getTablePath(DATA1_TABLE_ID)).isEmpty();
+        assertThat(serverMetadataCache.getTableInfo(DATA1_TABLE_ID)).isEmpty();
+        assertThat(serverMetadataCache.getTableMetadata(DATA1_TABLE_PATH)).isEmpty();
+
+        // test delete one partition.
+        serverMetadataCache.updateClusterMetadata(
+                new ClusterMetadata(
+                        coordinatorServer,
+                        aliveTableServers,
+                        Collections.emptyList(),
+                        Collections.singletonList(
+                                new PartitionMetadata(
+                                        partitionTableId,
+                                        partitionName1,
+                                        DELETED_PARTITION_ID, // mark this partition as
+                                        // deletion.
+                                        Collections.emptyList()))));
+        assertThat(serverMetadataCache.getPartitionName(partitionId1)).isEmpty();
+        assertThat(
+                        serverMetadataCache.getPartitionMetadata(
+                                PhysicalTablePath.of(partitionedTablePath, partitionName1)))
+                .isEmpty();
+        assertPartitionMetadataEquals(
+                partitionId2,
+                partitionTableId,
+                partitionId2,
+                partitionName2,
+                Arrays.asList(
+                        new BucketMetadata(0, 0, 0, Arrays.asList(0, 1, 2)),
+                        new BucketMetadata(1, NO_LEADER, 0, Arrays.asList(1, 0, 2))));
```

Comment:
There are toooo many BucketMetadatas, could you declare some reusable static BucketMetadata and just reference them in the test? Otherwise, it's hard to distinguish what's the difference between the changes.